src/store/modules/stock/actions.ts

```diff
@@ -26,6 +28,56 @@ const actions: ActionTree<StockState, RootState> = {
       logger.error(err)
       showToast(translate('No data available!'))
     }
+  },
+
+  async fetchInventoryCount({ commit }, { productId }) {
+    try {
+      const params = {
+        "entityName": "ProductFacility",
+        "inputFields": {
+          productId,
+          "facilityId": this.state.user.currentFacility.facilityId
+        },
+        "fieldList": ["minimumStock", "computedLastInventoryCount"],
+        "viewSize": 1
+      } as any
+      
+      const resp: any = await StockService.getInventoryComputation(params);
+      if(!hasError(resp)) {
+        commit(types.STOCK_ADD_PRODUCT_INFORMATION, { productId: productId, facilityId: this.state.user.currentFacility.facilityId, minimumStock: resp.data.docs[0].minimumStock, onlineAtp: resp.data.docs[0].computedLastInventoryCount })
+      } else {
+        throw resp.data;
+      }
+    }
+    catch (err) {
+      logger.error(err)
+      showToast(translate('No data available!'))
```

Comment:
We should not add a toast in this case, as we are already displaying `0` as a value for this fields if not present.